css-two-column-layout/styles.css

```diff
@@ -0,0 +1,118 @@
+.container {
+  max-width: 700px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.grayfont {
+  color: #a1a1a1;
+}
+
+.column {
+  width: 100%;
+}
+
+.center {
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+}
+
+h1 {
+  font-family: Karla, sans-serif;
+  font-weight: 600;
+  text-transform: uppercase;
+  margin-bottom: 0;
+  word-spacing: 0.5rem;
+  letter-spacing: 0.25rem;
+  width: 100%;
+  font-style: thin;
+}
+
+p {
+  font-family: Karla, sans-serif;
+  line-height: 1.25rem;
+  font-size: 70%;
+}
+
+button {
+  background-color: white;
+  border-color: #dd7a76;
+  padding: 0.4rem 1.75rem;
+  font-family: Karla, sans-serif;
+  font-size: 70%;
+  text-transform: uppercase;
+  color: #dd7a76;
+  border-width: 1.5px;
+}
+
+.down {
+  display: flex;
+  flex-direction: column;
+}
+
+.smaller {
+  width: 460px;
+}
+
+.row2 {
+  font-family: Karla, sans-serif;
+}
```

Comment:
1. Do not number your classes, it gives no insight into their meaning or usage.
2. You should just make this a utility class to apply font family.